app/containers/ItunesContainer/reducer.js

```diff
@@ -0,0 +1,49 @@
+/*
+ *
+ * itunesContainer reducer
+ *
+ */
+
+import produce from 'immer';
+import { createActions } from 'reduxsauce';
+
+export const initialState = {
+  searchTerm: null,
+  gridData: {},
+  searchError: null
+};
+
+export const { Types: itunesContainerTypes, Creators: itunesContainerCreators } = createActions({
+  searchItunes: ['searchTerm'],
+  successSearchItunes: ['data'],
+  failureSearchItunes: ['error'],
+  clearGridData: {}
+});
```

Comment:
can we give better naming to these

requestGetTracks
successGetTracks
failureGetTracks
clearTracks

Reply:
Yes, updated the variable names accordingly